components/aws/sagemaker/batch_transform/component.yaml

```diff
@@ -2,21 +2,101 @@ name: 'SageMaker - Batch Transformation'
 description: |
   Batch Transformation Jobs in SageMaker
 inputs:
-  - {name: region,          description: 'The region where the cluster launches.'}
-  - {name: model_name,      description: 'The name of the model that you want to use for the transform job.'}
-  - {name: input_location,  description: 'The S3 location of the data source that is associated with a channel.'}
-  - {name: output_location, description: 'The Amazon S3 path where you want Amazon SageMaker to store the results of the transform job.'}
+  - name: region
+    description: 'The region where the cluster launches.'
+  - name: job_name
+    description: 'The name of the batch transform job.'
+    default: ''
```

Comment:
Does it mean user always need to explicitly specify a job name?

Reply:
Nope, they can omit this param from their pipeline or leave it empty and it will default to a job name the way it was done before